server/.golangci.yml

```diff
@@ -42,6 +42,10 @@ issues:
       text: "var-naming|error-naming|exported|increment-decrement|error-strings|if-return|unused-parameter|blank-imports|empty-block"
       # We need to fix the unused parameter issues and remove the exception.
 
+    - linters:
+      - revive
+      text: "should replace|stutters|underscore|annoying|error strings should not be capitalized"
```

Comment:
I don't think this is right. First of all, this block is just duplicated from above. And secondly, we are adding more ignore strings when we want to reduce the ignore strings.

When https://github.com/mattermost/mattermost/issues/16623 was created, golint was our linter. But now we use, revive, so things might have changed slightly. You have to check the documentation for revive, and see what is the error message that is highlighted for these intialism errors. And then ignore the errors only for the `channels/store/storetest` directory.